database/buildSchema/26_review_assignment.sql

```diff
@@ -1,12 +1,17 @@
 -- review assignment
 
+CREATE TYPE public.review_assignment_status as ENUM ('Available', 'Not available', 'Assigned', 'Available for self assignment'); 
+
 CREATE TABLE public.review_assignment (
 	id serial primary key,
 	assigner_id integer references public.user(id),
 	reviewer_id integer references public.user(id),
 	stage_id integer references public.template_stage(id),
+	"status" public.review_assignment_status,
 	application_id integer references public.application(id),
-	trigger public.trigger
+	available_sections_ids integer [],
```

Comment:
Is it too much to call it `availableTemplateSectionIds`? (Just that it's not immediately clear if it means template section or application section)

Reply:
Sure. I can change to that - will also change on the insert data `05_review_assignments` file